playground/src/CodeEditor.tsx

```diff
@@ -2,21 +2,32 @@ import * as React from 'react';
 import * as monacoEditor from 'monaco-editor';
 import { FlexColDiv } from './FlexDivs';
 
-export interface ICommentSyntaxMarker {
-  message: string;
-
+export interface ISyntaxLocation {
```

Comment:
>ISyntaxLocation [](start = 17, length = 15)

The compiler has a type ITextRange with an identical signature.  I think it's a better name, since a location would be a single index, not a start/end pair